### PR TITLE
Sara/sms 354 update docs with scan detail page

### PR DESCRIPTION
**Pull request by s-santillan**
# Thanks for improving Semgrep Docs 😀

## About this PR

[Preview link](https://deploy-preview-1794--semgrep-docs-prod.netlify.app/docs/deployment/manage-projects)

We don't have a page explaining the Projects page at all, but over time the Projects page has become a more comprehensive & mature feature that I think sits under deployment, so I made a page for projects, and under that, more specific information about the latest scan details page. I focused on the value of the page as a place to inspect for set up and troubleshooting tasks.

### Please ensure

- [x] A subject matter expert (SME) reviews the content
- [x] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team
- [x] Redirects are added if the PR changes page URLs
- [x] Go through managed scans documentation to check steps to view scan logs

### CODE REVIEW

**Review by sabrinabrogren — APPROVED** [review]
Awesome, looks good to me!

File: docs/deployment/manage-projects.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+hide_title: true
+description: View projects, detailed logs and information for any scan.
+tags:
+    - Deployment
+    - Semgrep AppSec Platform
+---
+
+# Manage projects
+
+View, sort, and tag your projects through the **Projects** page. A **project** is any repository that you have scanned with Semgrep AppSec Platform through either CI or Semgrep Managed Scans, and includes that repository's findings and metadata.
```
Comment (khorne3) [inline docs/deployment/manage-projects.md:13]:
```suggestion
View, sort, and tag your projects through the **Projects** page. A **project** is any repository that you have scanned with Semgrep AppSec Platform through CI or Semgrep Managed Scans, and includes that repository's findings and metadata.
```

File: docs/deployment/manage-projects.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+1. Navigate to [Semgrep AppSec Platform](https://semgrep.dev/login) and click **<i class="fa-solid fa-folder-open"></i> Projects**.
+1. In the row of the project you want to view, click the project's **<i class="far fa-window-restore"></i> window icon** under the **Details** column.
+
+The following filters are available:
+
+- **Time period**: 7 days or 1 month
+- **Scan type**: Full or diff-aware scans
+- **Status**: Running, completed, error, or never finished
+- **Duration**: You can set the amount of hours or minutes
```
Comment (khorne3) [inline docs/deployment/manage-projects.md:51]:
```suggestion
- **Duration**: The amount of time the scan took to complete in hours or minutes
```

File: docs/deployment/manage-projects.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    - Semgrep AppSec Platform
+---
+
+# Manage projects
+
+View, sort, and tag your projects through the **Projects** page. A **project** is any repository that you have scanned with Semgrep AppSec Platform through either CI or Semgrep Managed Scans, and includes that repository's findings and metadata.
+
+Refer to this page to **manage** and **troubleshoot** thousands of repositories by identifying scan issues or scans with a high finding count.
+
+Additionally, the Semgrep API enables you to filter tags for additional workflows and integrations into your own systems. Create tags based on engineering or department teams, external-facing or internal codebases, and so on. See [Tags](/semgrep-appsec-platform/tags) for more information.
```
Comment (khorne3) [inline docs/deployment/manage-projects.md:17]: This paragraph feels a bit out of place, so maybe move it to the end of the section / just above the Sort projects section.

**Review by khorne3 — APPROVED** [review]

**Pull request merged**